packages/misc/src/lib/misc.ts

```diff
@@ -210,16 +213,13 @@ export const throwGenericError = (e: any): never => {
 export const isNodeClientErrorV1 = (
   nodeError: NodeClientErrorV0 | NodeClientErrorV1
 ): nodeError is NodeClientErrorV1 => {
-  return (
-    nodeError.hasOwnProperty('errorCode') &&
-    nodeError.hasOwnProperty('errorKind')
-  );
+  return 'errorCode' in nodeError && 'errorKind' in nodeError;
```

Comment:
Using VError, we would use `VError.meta()` to get a flattened object, then continue to use `hasOwnProperty` https://www.npmjs.com/package/@openagenda/verror#verrormetaerr`.  Or even better, we could just check `name` of the error.  Generally, all errors should have a `name` and that it a reasonable way to identify a particular type of error that is less brittle than using `instanceof`

Reply:
This is removed in the PR https://github.com/LIT-Protocol/js-sdk/pull/576 so it would not apply anymore
Lucky us, it was only used in `throwError`